public/scripts/statusDisplay.js

```diff
@@ -0,0 +1,35 @@
+const displayDiv = document.querySelector(".display-div");
+
+export const updateStatusDisplay = (state) => {
+  let displayStatusMessage = displayDiv.innerText;
```

Comment:
Instead of holding a reference to this variable, why not just assign `displayDiv.innerText` within each conditional?

Reply:
I'm not sure why that didn't occur to me, but I've made the change to avoid needlessly holding a reference to the variable.